airbyte-integrations/connectors/source-chargify/source_chargify/schemas/invoices.json

```
@@ -0,0 +1,501 @@
+            }
+        },
+        "taxes" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties" : {
+                    "uid" : {
+                        "type" : "string"
+                    },
+                    "title" : {
+                        "type" : "string"
+                    },
+                    "source_type" : {
+                        "type" : "string"
+                    },
+                    "source_id" : {
+                        "type" : "integer"
+                    },
+                    "percentage" : {
+                        "type" : "string"
+                    },
+                    "taxable_amount" : {
+                        "type" : "string"
+                    },
+                    "discount_amount" : {
+                        "type" : "string"
+                    }
+                }
+            }
+        },
+        "credits" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties" : {
+                    "uid" : {
+                        "type" : "string"
+                    },
+                    "credit_note_number" : {
+                        "type" : "string"
+                    },
+                    "credit_note_uid" : {
+                        "type" : "string"
+                    },
+                    "transaction_time" : {
+                        "type" : "string"
+                    },
+                    "memo" : {
+                        "type" : "string"
+                    },
+                    "original_amount" : {
+                        "type" : "string"
+                    },
+                    "applied_amount" : {
+                        "type" : "string"
+                    }
+                }
+            }
+        },
+        "refunds" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties" : {
+                    "transaction_id" : {
+                        "type" : "integer"
+                    },
+                    "payment_id" : {
+                        "type" : "integer"
+                    },
+                    "memo" : {
+                        "type" : "string"
+                    },
+                    "original_amount" : {
+                        "type" : "string"
+                    },
+                    "applied_amount" : {
+                        "type" : "string"
+                    },
+                    "gateway_transaction_id" : {
+                        "type" : ["string", "null"]
+                    }
+                }
+            }
+        },
+        "payments" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties" : {
+                    "transaction_time" : {
+                        "type" : "integer"
```

[Endpoint docs](https://developers.chargify.com/docs/api-docs/aa24a44561ce2-list-invoices#response-body) say `string`, but can this data fit into integer?
```suggestion
                        "type" : "string"
```